src/common/serialize.rs

```diff
@@ -14,7 +14,20 @@ pub trait BinarySerializable: fmt::Debug + Sized {
     fn deserialize<R: Read>(reader: &mut R) -> io::Result<Self>;
 }
 
-/// `FixedSize` marks a `BinarySerializable` as
```

Comment:
Did you remove this comment line on purpose?

Reply:
that was by accident